src/App.css

```diff
@@ -1,22 +1,28 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
 }
-
-.App-logo {
-  height: 40vmin;
+ol,
+ul {
```

Comment:
Be careful applying global styles because this means that ALL lists won't be styled throughout the app. It may be useful to use a className instead just so that we minimize the scope of this styling.